### ISSUE TITLE
Set link alt text to the macro url for easier debugging in browser

### ISSUE THREAD

**Issue by invicticide** [invicticide/fractive#27]
For normal links this is no big deal because the target url appears in the browser's status bar.

For macro links this sucks because they just have `href=#` since their click is handled in Javascript. From a player perspective, nobody cares, but from a dev perspective it'd be nice to be able to see the actual macro being linked to.

If we just set `href={@someSection}` the userAgent will probably try to resolve the `href` on click, which is *no bueno*. Instead, we can set `alt={@someSection}` and/or `title={@someSection}` so the macro target appears in a tooltip.

This might also be worth hiding behind some debug flag in the build configuration that doesn't exist yet, so it's omitted from "release" builds.

**Comment by invicticide** [comment]
Link alt texts will now be set when "linkTooltips" is set to true in the project's fractive.json. If it's false, no alt texts will be generated for macro links.
